autotest/ogr/ogr_gml_fgd_read.py

```diff
@@ -0,0 +1,105 @@
+###############################################################################
+# $Id$
+#
+# Project:  GDAL/OGR Test Suite
+# Purpose:  GML Reading Driver for Japanese FGD GML v4 testing.
+# Author:   Frank Warmerdam <warmerdam@pobox.com>
+#
+###############################################################################
+# Copyright (c) 2006, Frank Warmerdam <warmerdam@pobox.com>
+# Copyright (c) 2008-2014, Even Rouault <even dot rouault at mines-paris dot org>
+# Copyright (c) 2017, Hiroshi Miura <miurahr@linux.com>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+###############################################################################
+
+import os
+import sys
+
+sys.path.append( '../pymod' )
+
+import gdaltest
+import ogrtest
+from osgeo import gdal
+from osgeo import ogr
+from osgeo import osr
+import shutil
+
+
+###############################################################################
+# Test reading Japanese FGD GML (v4) files
+###############################################################################
+
+_fgd_dir = 'data/gml_jpfgd/'
+
+###############################################################################
+# Test reading Japanese FGD GML (v4) ElevPt file
+
+def ogr_gml_fgd_1():
+
+    if not gdaltest.have_gml_reader:
+        return 'skip'
+
+    ### open FGD GML file
+    ds = ogr.Open(_fgd_dir + 'ElevPt.xml')
+
+    # check number of layers
+    if ds.GetLayerCount() != 1:
+        gdaltest.post_reason('Wrong layer count')
+        return 'fail'
+
+    lyr = ds.GetLayer(0)
+
+    # check the SRS
+    sr = osr.SpatialReference()
+    sr.ImportFromEPSG(6668)   # JGD2011
+    if not sr.IsSame(lyr.GetSpatialRef()):
+        gdaltest.post_reason('Wrong SRS')
+        return 'fail'
+
+    # check the first feature
+    feat = lyr.GetNextFeature()
+    if ogrtest.check_feature_geometry(feat, 'POINT (133.123456789 34.123456789)'):
+        gdaltest.post_reason('Wrong geometry')
+        return 'fail'
+
```

Comment:
Not all test fails, but some pass. It is strange. I'd like to investigate it in detail.